fabric_cf/actor/core/policy/broker_simpler_units_policy.py

```diff
@@ -412,31 +428,66 @@ def ticket(self, *, reservation: IBrokerReservation, node_id_to_reservations: di
 
         return False, node_id_to_reservations
 
+    def __candidate_nodes(self, sliver: NodeSliver) -> List[str]:
+        """
+        Identify candidate worker nodes in this site that have at least
+        as many needed components as in the sliver.
+        """
+        node_props = {'Site': sliver.site, 'Type': str(NodeType.Server)}
```

Comment:
Can we replace with ABCPropertyGraphConstants.PROP_SITE and ABCPropertyGraphConstants.PROP_TYPE? This is my fault.